src/pages/Lending/hooks/useLendingCloseQuery.ts

```diff
@@ -236,24 +226,22 @@ export const useLendingQuoteCloseQuery = ({
       selectLendingCloseQueryData({
         data,
         collateralAssetMarketData,
-        repaymentAmountCryptoPrecision,
-        repaymentAmountFiatUserCurrency,
+        repaymentAssetMarketData,
         repaymentPercent,
       }),
     // Do not refetch if consumers explicitly set enabled to false
     // They do so because the query should never run in the reactive react realm, but only programmatically with the refetch function
-    refetchIntervalInBackground: enabled,
-    refetchInterval: enabled ? 20_000 : undefined,
+    refetchIntervalInBackground: true,
+    refetchInterval: 20_000,
```

Comment:
👋🏽 we *always* want to refetch and don't want to introduce a mechanism where this could be skipped